conversions/conversions.go

```diff
@@ -23,6 +23,23 @@ var factors = map[string]float64{
 	constants.UnitKimometres:    constants.EarthRadius / 1000.0,
 }
 
+var areaFactors = map[string]float64{
+	constants.UnitAcres:       0.000247105,
+	constants.UnitCentimeters: 10000.0,
+	constants.UnitCentimetres: 1000.0,
```

Comment:
I think the value for `constants.UnitCentimetres` should have been the same as that of `constants.UnitCentimeters`, correct?

Reply:
Did you change this?

Reply:
Yeap